content/en/docs/contribute/localization.md

```diff
@@ -361,6 +361,20 @@ extensive human review to meet minimum standards of quality.
 To ensure accuracy in grammar and meaning, members of your localization team
 should carefully review all machine-generated translations before publishing.
 
+### Translating SVG Images
```

Comment:
```suggestion
### Localizing images
```

Reply:
Hmm, I believe it would be more appropriate to name this subsection **Translating SVG Images** instead of **Localizing**. Since the outer section title is **Translating Content**.